src/Projection/SettingProjector.ts

```diff
@@ -7,6 +7,7 @@ import { SimpleSetting } from '../Domain/Setting/SimpleSetting'
 export class SettingProjector {
   async projectSimple(setting: Setting): Promise<SimpleSetting> {
     const { 
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
       user,
```

Comment:
does changing to `_user` fix the issue?

Reply:
nope
to be precise, it would have to be `user: _user` -- this doesn't work either

Reply:
What about just an underscore?

Reply:
Doesn't work and would not handle for multiple ignored values.

I tried different combinations of these before entering the disable lines. None of them work.

Reply:
Probably the `user: _user` can be made to work with some linter rule -- will check and add if I find one.
Anyway I didn't think it was that important.